tests/e2e/framework/projutil.go

```diff
@@ -133,11 +133,12 @@ func (opts GoCmdOptions) setCmdFields(c *exec.Cmd) {
 }
 
 // From https://github.com/golang/go/wiki/Modules:
-//	You can activate module support in one of two ways:
-//	- Invoke the go command in a directory with a valid go.mod file in the
-//      current directory or any parent of it and the environment variable
-//      GO111MODULE unset (or explicitly set to auto).
-//	- Invoke the go command with GO111MODULE=on environment variable set.
+//
+//		You can activate module support in one of two ways:
+//		- Invoke the go command in a directory with a valid go.mod file in the
+//	     current directory or any parent of it and the environment variable
+//	     GO111MODULE unset (or explicitly set to auto).
+//		- Invoke the go command with GO111MODULE=on environment variable set.
```

Comment:
I need to revert this since it's not related to this change and it's being fixed separately in a different commit.

https://github.com/ComplianceAsCode/compliance-operator/pull/266